From david.vrabel@csr.com  Thu Jun 19 11:58:54 2008
From: David Vrabel <david.vrabel@csr.com>
Date: Wed, 11 Jun 2008 12:04:41 +0100
Subject: UWB: make UWB selectable on all archs with USB support
To: Byron Bradley <byron.bbradley@gmail.com>
Cc: Andrew Morton <akpm@linux-foundation.org>, linux-kernel@vger.kernel.org, linux-mm@kvack.org, Greg KH <greg@kroah.com>
Message-ID: <484FB149.4080000@csr.com>


ARM (and some other architectures) don't use drivers/Kconfig.  This
patch enables UWB on all these architectures that have USB support.

Reported-by: Byron Bradley <byron.bbradley@gmail.com>
Signed-off-by: David Vrabel <david.vrabel@csr.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


---
 arch/arm/Kconfig   |    2 ++
 arch/cris/Kconfig  |    2 ++
 arch/h8300/Kconfig |    2 ++
 arch/v850/Kconfig  |    2 ++
 4 files changed, 8 insertions(+)

--- a/arch/arm/Kconfig
+++ b/arch/arm/Kconfig
@@ -1169,6 +1169,8 @@ source "drivers/hid/Kconfig"
 
 source "drivers/usb/Kconfig"
 
+source "drivers/uwb/Kconfig"
+
 source "drivers/mmc/Kconfig"
 
 source "drivers/leds/Kconfig"
--- a/arch/cris/Kconfig
+++ b/arch/cris/Kconfig
@@ -677,6 +677,8 @@ source "fs/Kconfig"
 
 source "drivers/usb/Kconfig"
 
+source "drivers/uwb/Kconfig"
+
 source "arch/cris/Kconfig.debug"
 
 source "security/Kconfig"
--- a/arch/h8300/Kconfig
+++ b/arch/h8300/Kconfig
@@ -227,6 +227,8 @@ source "drivers/hwmon/Kconfig"
 
 source "drivers/usb/Kconfig"
 
+source "drivers/uwb/Kconfig"
+
 endmenu
 
 source "fs/Kconfig"
--- a/arch/v850/Kconfig
+++ b/arch/v850/Kconfig
@@ -342,6 +342,8 @@ source "sound/Kconfig"
 
 source "drivers/usb/Kconfig"
 
+source "drivers/uwb/Kconfig"
+
 source "arch/v850/Kconfig.debug"
 
 source "security/Kconfig"
